server/command.go

```diff
@@ -428,6 +432,10 @@ func (p *Plugin) executeConnectBotCommand(args *model.CommandArgs) (*model.Comma
 		return p.cmdError(args.UserId, args.ChannelId, "Unable to connect the bot account, only system admins can connect the bot account.")
 	}
 
+	if storedToken, _ := p.store.GetTokenForMattermostUser(p.userID); storedToken != nil {
+		return p.cmdError(args.UserId, args.ChannelId, "The bot account is already connected to MS Teams. Please disconnect the bot account first before connecting again.")
```

Comment:
I think we should create the constant for this as this check will be used many times in the code.

Reply:
We should create a constant for what? The error message?